lib/tests/test_lexer.c

```diff
@@ -312,11 +312,11 @@ Test(lexer, at_version_stores_config_version_in_parsed_version_in_hex_form)
   parser->lexer->ignore_pragma = FALSE;
 
   cfg_set_version_without_validation(configuration, 0);
-  _input("@version: 3.28\n\
+  _input("@version: current\n\
 foo\n");
   assert_parser_identifier("foo");
-  cr_assert_eq(configuration->user_version, 0x031c,
-               "@version parsing mismatch, value %04x expected %04x", configuration->user_version, 0x031c);
+  cr_assert_eq(configuration->user_version, VERSION_VALUE_CURRENT,
+               "@version parsing mismatch, value %04x expected %04x", configuration->user_version, VERSION_VALUE_CURRENT);
 
```

Comment:
with this change, this test would not test what its intention was. To parse a version number in hex form. If we want, we can add a new testcase for `@version: current`

Reply:
here is the whole context:
```
Test(lexer, at_version_stores_config_version_in_parsed_version_in_hex_form)
{
  parser->lexer->ignore_pragma = FALSE;

  cfg_set_version_without_validation(configuration, 0);
  _input("@version: 3.28\n\
foo\n");
  assert_parser_identifier("foo");
  cr_assert_eq(configuration->user_version, 0x031c,
               "@version parsing mismatch, value %04x expected %04x", configuration->user_version, 0x031c);

  cfg_set_version_without_validation(configuration, 0);
  _input("@version: 3.1\n\
bar\n");
  assert_parser_identifier("bar");
  cr_assert_eq(configuration->user_version, 0x0301,
               "@version parsing mismatch, value %04x expected %04x", configuration->user_version, 0x0301);

  cfg_set_version_without_validation(configuration, 0);
  _input("@version: 3.5\n\
baz\n");
  assert_parser_identifier("baz");
  cr_assert_eq(configuration->user_version, 0x0305,
               "@version parsing mismatch, value %04x expected %04x", configuration->user_version, 0x0305);
}

```

There is a duplication in the test for testing the `current` version.
I can modify it, but this is independent from this PR: we have a case where we want to test the current version...